info/v2/container.go

```diff
@@ -85,7 +85,7 @@ type ContainerSpec struct {
 	Image string `json:"image,omitempty"`
 }
 
-type ContainerStats struct {
+type DeprecatedContainerStats struct {
```

Comment:
As mentioned offline, go with whatever you think will be easiest in terms of unblocking the critical parts. If you think you can get the critical part merged faster by splitting this up, go with that. Otherwise, I'm fine with leaving it as is.